src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.OSX.cs

```diff
@@ -160,7 +164,7 @@ private void CreateStreamAndStartWatcher()
                     _callback,
                     arrPaths,
                     Interop.EventStream.kFSEventStreamEventIdSinceNow,
-                    0.2f,
+                    0.0f,
```

Comment:
This value didn't seem to cause a difference in coalescing...the OS still coalesced events as expected. I'm not 100% sure what this value is supposed to affect since changing it to 0 made no noticeable difference; due to that, making it 0 is much more clear that it isn't some magic value